docs/user-guide/dream/dream-advanced-powder-reduction.ipynb

```diff
@@ -453,7 +461,134 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "result.hist(tof=300).plot()"
+    "histograms = []\n",
+    "for events in result.values():\n",
+    "    hist = events.hist()\n",
+    "    # Apply the mask\n",
+    "    if (mask := hist.masks.pop('zero_vanadium', None)) is not None:\n",
+    "        hist.data = sc.where(mask, sc.scalar(0.0), hist.data)\n",
+    "    histograms.append(hist)\n",
+    "\n",
+    "sc.concat(histograms, dim='detector').sum('detector').plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "35",
+   "metadata": {},
+   "source": [
+    "### 2D intensity vs. d-spacing and 2θ\n",
+    "\n",
+    "Just like before, we map the workflow over the detector names.\n",
+    "But here we also assign different $2\\theta$ bins for each detector bank."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "36",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "detector_names = [\"mantle\", \"endcap_forward\", \"endcap_backward\", \"high_resolution\"]\n",
+    "two_theta_bins = [\n",
+    "    sc.linspace(dim=\"two_theta\", unit=\"rad\", start=0.77, stop=2.36, num=201),\n",
+    "    sc.linspace(dim=\"two_theta\", unit=\"rad\", start=0.24, stop=0.71, num=101),\n",
+    "    sc.linspace(dim=\"two_theta\", unit=\"rad\", start=2.42, stop=2.91, num=151),\n",
+    "    sc.linspace(dim=\"two_theta\", unit=\"rad\", start=2.94, stop=3.11, num=51),\n",
```

Comment:
I find it annoying that we have to know all these values.
We figured it out by plotting the data for each one, but it's difficult for a user (instrument scientist) to know theme without plotting first...

Reply:
True. Should we compute them based on geometry and store them in the package? To do this precisely, we would need the voxel positions and sized. The simulation only has event (sub-voxel) positions.

Reply:
I would have hoped we could just use a constant bin width for everyone, and let the binning/histogramming figure out the bounds?

Reply:
For the record: We tried to use a common number of bins or a common bin width instead of mapping over edges. This leads to different binning in sample and vanadium data which means that `_normalize_by_vanadium` needs to be adjusted. Doing so is non trivial because it has to deal with 2D data. Since in practice, we expect people to use specific, known-to-be-good bin edges that are determined by scientists, we stick with the current implementation of mapping over edges.